pkg/workloadmeta/collectors/podman/podman.go

```diff
@@ -157,9 +158,42 @@ func (c *collector) expiredEvents() []workloadmeta.CollectorEvent {
 func networkIPs(container *podman.Container) map[string]string {
 	res := make(map[string]string)
 
-	if len(container.State.NetworkStatus) > 0 {
-		if len(container.State.NetworkStatus[0].IPs) > 0 {
-			res["podman"] = container.State.NetworkStatus[0].IPs[0].Address.IP.String()
+	// container.Config.Networks contains only the networks specified at container creation time
+	// and not the ones attached afterwards with `podman network attach`
+	// They appear in the order in which they were specified in the `podman run --net=…` command
+	networkNames := make([]string, len(container.Config.Networks))
```

Comment:
I think that when no network is specified with `--net`, `container.Config.Networks` contains 0 elements.
But `container.State.NetworksStatus will have one element that we should associate with the default network ("podman").

Reply:
I think that if `len(container.Config.Networks) == 0`, we should probably just append here "podman" so the logic below that compares lengths still works.